packages/fast-components-styles-msft/src/utilities/typography.ts

```diff
@@ -105,3 +110,32 @@ export function applyTypeRampConfig(typeConfig: keyof TypeRamp): CSSRules<Design
         lineHeight: toPx(typeRamp[typeConfig].lineHeight),
     };
 }
+
+/**
+ * Adds the font size and line height attributes for the type ramp, scaled according to the design system density config.
+ *
+ * @param config The design system config.
+ * @param typeConfig The default type ramp config at base density.
+ */
+export function scaleApplyTypeRampConfigWithDensity(
```

Comment:
I know you guys talked about this a bit, but I think something like `scaleTypography` may be better to use in development. I also think we want something like this without passing the design system:
```
scaleTypography("t7")
```

should return:
```
{
   fontSize: (designSystem: DesignSystem): string => {},
   lineHeight: (designSystem: DesignSystem): string => {},
}
```

If need be, we can do this as a follow-up - although, I think we should land the name...that way we're just changing internals.